webpack.config.js

```diff
@@ -1,13 +1,13 @@
 const path = require('path');
-const ExtractTextPlugin = require("extract-text-webpack-plugin");
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 module.exports = {
   entry: './src/client/index.js',
   output: {
     path: path.join(__dirname, 'public'),
     publicPath: '/',
     filename: 'bundle.js',
-    chunkFilename: 'bundle.js'
+    chunkFilename: 'bundle.js',
```

Comment:
hey @jcelder, does this cause a linter error?

Reply:
just kidding, the linter appears to be adding trailing commas (and changing double quotes to single quotes). i am okay with this... is the rest of the team, too? xD